echidna-lb/src/config/mod.rs

```diff
@@ -0,0 +1,47 @@
+use clap::Parser;
+use serde::Deserialize;
+use serde_yaml::Error;
+use std::fs;
+use std::path::Path;
+
+#[derive(Parser)]
+#[clap(author, version, about, long_about = None)]
+pub struct Args {
+    /// Path to the configuration file
+    #[clap(short, long, default_value = "./config.yaml")]
+    pub config: String,
```

Comment:
Maybe use a PathBuf instead?
```suggestion
    pub config: std::path::PathBuf,
```